sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -76,7 +76,15 @@ case class CurrentTimestamp() extends LeafExpression with CodegenFallback {
  */
 @ExpressionDescription(
   usage = "_FUNC_(start_date, num_days) - Returns the date that is num_days after start_date.",
-  extended = "> SELECT _FUNC_('2016-07-30', 1);\n '2016-07-31'")
+  extended = """
+    Arguments:
+      start_date - a date expression.
+      num_days - a numeric expression that represents the number of days to add.
```

Comment:
Then, in this PR, I saw multiple arguments are using `integral expressions`. Casting is not using?